.github/workflows/deploy.yaml

```diff
@@ -6,24 +6,24 @@ on:
       - master
 
 jobs:
-  agents:
-    if: ${{ !startsWith(github.event.head_commit.message, 'Release ') }}
-    runs-on: ubuntu-latest
-    name: Agent 1
-    timeout-minutes: 60
-    strategy:
-      matrix:
-        agent: [ 1, 2, 3 ]
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - uses: actions/setup-node@v1
-        with:
-          node-version: '14'
-      - run: yarn
-      - name: Start Nx Agent ${{ matrix.agent }}
-        run: npx nx-cloud start-agent
+  #  agents:
```

Comment:
don't we need these commented jobs and configs?

Reply:
not my change, not sure